test-suite workflow failure

**Describe the bug**
The `test-suite` workflow sometimes fail because a port is in use.

```
INFO[2022-05-31T02:11:18Z]route.go:116 listenAndServe [route] Internal ListenAndServe              
INFO[2022-05-31T02:11:18Z]route.go:120 listenAndServe [route] External ListenAndServe              
PANI[2022-05-31T02:11:18Z]tlsserver.go:88 ListenAndServe [tlsserver] listen failed: listen tcp :56002: bind: address already in use 
panic: (*logrus.Entry) 0xc00031c1c0

goroutine 25 [running]:
github.com/sirupsen/logrus.(*Entry).log(0xc00031c150, 0x0, 0xc000032320, 0x4a)
	/home/runner/work/edge-home-orchestration-go/edge-home-orchestration-go/vendor/github.com/sirupsen/logrus/entry.go:259 +0x2e5
github.com/sirupsen/logrus.(*Entry).Log(0xc00031c150, 0xc000000000, 0xc00010df68, 0x3, 0x3)
	/home/runner/work/edge-home-orchestration-go/edge-home-orchestration-go/vendor/github.com/sirupsen/logrus/entry.go:293 +0x86
github.com/sirupsen/logrus.(*Logger).Log(0xc0000fa070, 0x0, 0xc00010df68, 0x3, 0x3)
	/home/runner/work/edge-home-orchestration-go/edge-home-orchestration-go/vendor/github.com/sirupsen/logrus/logger.go:198 +0x7e
github.com/sirupsen/logrus.(*Logger).Panic(...)
	/home/runner/work/edge-home-orchestration-go/edge-home-orchestration-go/vendor/github.com/sirupsen/logrus/logger.go:247
github.com/lf-edge/edge-home-orchestration-go/internal/restinterface/route/tlsserver.(*TLSServer).ListenAndServe(0xc000206b40, 0xc0002e0900, 0x6, 0xc84dc0, 0x0)
	/home/runner/work/edge-home-orchestration-go/edge-home-orchestration-go/internal/restinterface/route/tlsserver/tlsserver.go:88 +0x232
created by github.com/lf-edge/edge-home-orchestration-go/internal/restinterface/route.RestRouter.listenAndServe
	/home/runner/work/edge-home-orchestration-go/edge-home-orchestration-go/internal/restinterface/route/route.go:114 +0x205
FAIL	github.com/lf-edge/edge-home-orchestration-go/internal/restinterface/route	0.017s
FAIL
error: exit status 1
Error: Process completed with exit code 1.
```

--- a/.github/workflows/test-suite.yml
+++ b/.github/workflows/test-suite.yml
@@ -7,6 +7,7 @@ jobs:
   test-suite:
     strategy:
       fail-fast: false
+      max-parallel: 1
       matrix:
         os: [ubuntu-18.04, ubuntu-20.04]
     runs-on: ${{ matrix.os }}

